packages/charrua-core/charrua-core.0.5/url

```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/mirage/charrua-core/releases/download/v0.5/charrua-core-0.5.tbz"
```

Comment:
Oh I see it also adds the stuff in Changes.md and so on.

I had mailed mindy asking her to change it, but maybe we can do this and I'll add the other ones retroactively, even though I'll have to cheat on 0.1-0.3 since there is no topkg.